apps/client/src/features/control/playback/tap-button/TapButton.tsx

```diff
@@ -1,11 +1,11 @@
-import { ForwardedRef, forwardRef, PropsWithChildren } from 'react';
+import { ButtonHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react';
 import { Playback } from 'ontime-types';
 
 import { cx } from '../../../../common/utils/styleUtils';
 
 import style from './TapButton.module.scss';
 
-interface TapButtonProps {
+interface TapButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
cleanup: Since we extend the HTML Button attributes, we no longer need the normal stuff like `disabled` `onClick` and `className`